plugins/cxaudio/README.md

```diff
@@ -0,0 +1,44 @@
+Conexant Audio Support
+======================
+
+Introduction
+------------
+
+This plugin is used to update a small subset of Conexant (now owned by Synaptics)
+audio devices.
+
+Firmware Format
+---------------
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+a modified SREC file format.
+
+This plugin supports the following protocol ID:
+
+ * com.synaptics.cxaudio
+
+GUID Generation
+---------------
+
+These devices use the standard USB DeviceInstanceId values, e.g.
+
+ * `USB\VID_17EF&PID_3083&REV_0001`
+ * `USB\VID_17EF&PID_3083`
+ * `USB\VID_17EF`
+
+These devices also use custom GUID values, e.g.
+
+ * `CXAUDIO\CX2198X`
+ * `CXAUDIO\CX21985`
+
+Quirk Use
+---------
+
+This plugin uses the following plugin-specific quirks:
+
+| Quirk                      | Description                      | Minimum fwupd version |
+|----------------------------|----------------------------------|-----------------------|
+| `ChipIdBase`               | Base integer for ChipID          | 1.3.2                 |
```

Comment:
Is this like family?  Or it's literally an integer added to an offset to get a chip ID?

Reply:
Exactly the latter.